examples/custom_ct2_model.py

```diff
@@ -0,0 +1,144 @@
+import os
+import argparse
+from dataclasses import dataclass
+from functools import lru_cache
+from crossfit import op
+from crossfit.backend.torch.hf.model import HFModel
+import dask_cudf
+import crossfit as cf
+
+import ctranslate2
+from transformers import AutoConfig, AutoTokenizer
+
+@dataclass
+class TranslationConfig:
+    pretrained_model_name_or_path: str
+    ct2_model_path: str
+    string_tok_inf: bool = True
+
+class CT2CustomModel():
+    def __init__(self, config: TranslationConfig, device="cuda"):
+        self.config = config
+        self.tokenizer = AutoTokenizer.from_pretrained(
+            pretrained_model_name_or_path=config.pretrained_model_name_or_path,
+            trust_remote_code = True,
+        )
+        self.model = ctranslate2.Translator(model_path=config.ct2_model_path, device=device)
+        self.string_tok_inf = config.string_tok_inf
+
+    def clean_extra_tokens(self, token_2d):
+        results=[]
```

Comment:
```suggestion
        results = []
```
Can you run the `black` tool so that formatting mistakes like this will be fixed?